app/views/groups/show.html.slim

```diff
@@ -120,26 +103,6 @@
           .col-lg-6
             = render 'shared/summary_cards/main', resource: list
             
-  .row.group-details__row
+  .row.page-details__row
     .col-lg-12
-      .row.group-details__row--half
-        .col-lg-12
-          h3 DISCUSS
-      .row
-        .col-lg-12
-          = render 'comments/comment', resource: nil
-          = render 'comments/comment', resource: nil
-          = render 'comments/comment', resource: nil
-          = render 'comments/subcomment', resource: nil
-          = render 'comments/comment', resource: nil
-      .row.group-details__row
-        .col-lg-12.text-center
-          = link_to 'LOAD MORE COMMENTS', '#', class: 'btn btn-primary'
-          
-      .row
-        .col-lg-12
-          = form_tag '/', class: 'comment-form' do
-            .form-group
-              = text_area_tag :comment, '', class: 'form-control comment-form__textarea', placeholder: 'Add a comment...'
-            .form-group.pull-right
-              = button_tag 'SUBMIT', type: 'submit', class: 'form-control btn btn-primary'
+      = render 'shared/components/discuss'
```

Comment:
Excellent. This may be replaced with Disqus soon, so I'm glad it's in one place.